hub-detect/src/main/groovy/com/blackducksoftware/integration/hub/detect/workflow/report/DetailedSearchSummaryData.java

```diff
@@ -0,0 +1,24 @@
+package com.blackducksoftware.integration.hub.detect.workflow.report;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.blackducksoftware.integration.hub.detect.bomtool.BomTool;
+
+public class DetailedSearchSummaryData {
+    public String directory;
+    public List<BomToolSearchDetails> applicable;
+    public List<BomToolSearchDetails> notApplicable;
+    public List<BomToolSearchDetails> notSearchable;
+
+    public DetailedSearchSummaryData() {
+        applicable = new ArrayList<>();
+        notApplicable = new ArrayList<>();
+        notSearchable = new ArrayList<>();
+    }
+
+    public class BomToolSearchDetails {
+        public BomTool bomTool;
```

Comment:
We generally want to use private fields and getters/setters

Reply:
Changed.